MK3CHOPR/MK3CHOPR-IOC-01App/src/mk3Driver.cpp

```diff
@@ -66,8 +66,9 @@ mk3Driver::mk3Driver(const char *portName, const char *configFilePath, int mockC
     createParam(P_NominalFreqString, asynParamFloat64, &P_NominalFreq);
     createParam(P_NominalPhaseString, asynParamFloat64, &P_NominalPhase);
     createParam(P_NominalPhaseErrorString, asynParamFloat64, &P_NominalPhaseError);
-    createParam(P_DirectionEnabledString, asynParamInt32, &P_DirectionEnabled);
+    createParam(P_DirectionEnabledString, asynParamInt32, &P_ComputerMode);
     createParam(P_NumChannelsString, asynParamInt32, &P_NumChannels);
+    createParam(P_ComputerModeString, asynParamInt32, &P_NumChannels);
```

Comment:
P_NumChannels is being assigned twice, if this is correct please add a comment in the code as it looks a little strange. Likewise for using P_ComputerMode with DirectionEnabledString - is this for backward compatibility?